python/lsst/qserv/tests/benchmark.py

```diff
@@ -99,27 +115,38 @@ def getDatasetDir(testdata_dir, case_id):
         dataset_dir = os.path.join(testdata_dir, "case{0}".format(case_id))
         return dataset_dir
 
-    def runQueries(self, stopAt=MAX_QUERY):
-        self.logger.debug("Running queries : (stop-at: %s)", stopAt)
-        if self._mode == 'qserv':
+    def runQueries(self, mode, dbName, stopAt=MAX_QUERY):
+        """Run all queries agains loaded data.
+
+        Parameters
+        ----------
+        mode : str
+            One of "mysql" or "qserv"
```

Comment:
what's your philosophy about raising if `mode not in ('mysql', 'qserv')`?

Reply:
It cannot happen here :smile: but I will add a check and raise something.